stdlib/array.gr

```diff
@@ -62,10 +62,7 @@ let wrapNegativeIndex = (arrLen, idx) => {
  * @since v0.1.0
  */
 @unsafe
-provide let length = array => {
-  let ptr = WasmI32.fromGrain(array: Array<a>)
-  tagSimpleNumber(WasmI32.load(ptr, _ARRAY_LENGTH_OFFSET))
-}
+provide let length = array => length(array)
```

Comment:
Can't we just `provide { length }`?

Reply:
So we could do a `provide primitive length`
But then we don't get the argument name in the type signature.

Reply:
@spotandjake please open an issue about not having the argument name in the type signature and mentioned we can clean this up when that is fixed.

Reply:
This is now blocked on #1979

Reply:
@spotandjake this should be ready to rebase